jas/secrets/secretsscanner_test.go

```diff
@@ -98,15 +98,15 @@ func TestParseResults_ResultsContainSecrets(t *testing.T) {
 	secretScanManager.resultsFileName = filepath.Join(jas.GetTestDataPath(), "secrets-scan", "contain-secrets.sarif")
 
 	// Act
-	secretScanManager.secretsScannerResults, err = jas.ReadJasScanRunsFromFile(secretScanManager.resultsFileName, jfrogAppsConfigForTest.Modules[0].SourceRoot, secretsDocsUrlSuffix, severityutils.Medium)
+	vulnerabilitiesResults, _, err := jas.ReadJasScanRunsFromFile(secretScanManager.resultsFileName, jfrogAppsConfigForTest.Modules[0].SourceRoot, secretsDocsUrlSuffix, severityutils.Medium)
```

Comment:
Is this like the parallel func in iac and sast? does it checks for violations? does it check results. we can better clarify it in the func's name